src/devices/Sgp30/Sgp30Measurement.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Iot.Device.Sgp30
+{
+    /// <summary>
+    /// SGP30 Gss sensor measurement result.
+    /// </summary>
+    public struct Sgp30Measurement
+    {
+        /// <summary>
+        /// Total VOC result (parts per billion).
+        /// /// </summary>
+        public ushort Tvoc;

```

Comment:
Please use the UnitsNet. And you can adjust the comment giving the usual range of the sensor